src/components/pageheader/PageHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './pageheader.less';
+import { Typography } from 'antd';
+import styled from 'styled-components';
+const { Title, Paragraph } = Typography;
+
+interface PageHeaderProps {
+  readonly pageTitle: string;
+  readonly pageSubtitle: string;
+  readonly subtitleColor: string;
+}
+
+interface StyledSubtitleProps {
+  readonly textColor: string;
+}
+
+const StyledTitle = styled(Title)`
+  font-size: 44px;
+  line-height: 76px;
+  color: #3a681a;
+  font-weight: bold;
+  margin-bottom: 0px;
+`;
+
+const StyledSubtitle = styled(Paragraph)`
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 32px;
+  color: ${(props: StyledSubtitleProps) =>
```

Comment:
❓ Do we want the header color to be customizable? It's the same on all pages on Figma

Reply:
The main header color is constant but the subheader color has two different options on Figma (#444444 for logged-in landing and #3A681A for the leaderboard pages)